packages/cli/src/lib/compare/compare-command.ts

```diff
@@ -0,0 +1,36 @@
+import chalk from 'chalk';
+import { join } from 'node:path';
+import { CommandModule } from 'yargs';
+import { compareReportFiles } from '@code-pushup/core';
+import { PersistConfig } from '@code-pushup/models';
+import { CLI_NAME } from '../constants';
+import type { CompareOptions } from '../implementation/compare.model';
+import { yargsCompareOptionsDefinition } from '../implementation/compare.options';
+import { ui } from '../implementation/logging';
+
+export function yargsCompareCommandObject() {
+  const command = 'compare';
+  return {
+    command,
+    describe: 'Compare 2 report files and create a diff file',
+    builder: yargsCompareOptionsDefinition(),
```

Comment:
Your could add a group with name `Compare Options` to list all command options under the provided string instead of just 'Options'

Reply:
Sure. But I would only make that change once the other groups from #553 are merged.

Reply:
Also mentioned here, BTW :grin: https://github.com/code-pushup/cli/pull/556#discussion_r1521215217